CreeDictionary/utils/data_classes.py

```diff
@@ -0,0 +1,27 @@
+from typing import NamedTuple
+
+from utils.types import ConcatAnalysis
+
+
+class Analysis(NamedTuple):
+    """
+    Analysis of a wordform.
+    """
+
+    raw_prefixes: str
+    lemma: str
+    raw_suffixes: str
+
+    def concatenate(self) -> ConcatAnalysis:
+        result = ""
+        if self.raw_prefixes != "":
+            result += self.raw_prefixes + "+"
+        result += f"{self.lemma}+{self.raw_suffixes}"
+        return ConcatAnalysis(result)
+
+
+ORTHOGRAPHY_NAME = {
+    "Latn": "SRO (êîôâ)",
+    "Latn-x-macron": "SRO (ēīōā)",
+    "Cans": "Syllabics",
+}
```

Comment:
Good, so far, but why did this get moved here? Wouldn't `utils.vars` be better?

Reply:
This reminds me of TypedDicts/NamedTuple...but yeah, I'll move it to vars